torchvision/transforms/functional.py

```diff
@@ -18,6 +19,17 @@
 from . import functional_pil as F_pil
 from . import functional_tensor as F_t
 
+F = TypeVar("F", bound=Callable[..., Any])
```

Comment:
This is the [official way](https://mypy.readthedocs.io/en/stable/generics.html#declaring-decorators) to declare decorators that preserve the signature. Let's see if the test suite is still happy.